scipy/spatial/transform/rotation.py

```diff
@@ -616,3 +617,17 @@ def as_euler(self, seq, degrees=False):
             angles = np.rad2deg(angles)
 
         return angles[0] if self._single else angles
+
+    def inv(self):
+        """Returns the inverse of the current rotation
+
+        This function returns a new `Rotation` instance containing the inverse
+        of all the rotations in the current instance.
+        """
+        quat = self.as_quaternion().copy()
```

Comment:
Maybe it will be a little shorter and concise:
```
quat = self._quat.copy()
quat[:, -1] *= -1
if self._single:
   quat = quat[0]
return self.__class__(quat, normalized=True) (or self.from_quat(...)).
```